src/libs/blueprint/conduit_blueprint.hpp

```diff
@@ -60,6 +60,7 @@
 
 #include "conduit_blueprint_mesh.hpp"
 #include "conduit_blueprint_mesh_examples.hpp"
+#include "conduit_blueprint_mesh_examples_julia.hpp"
```

Comment:
@cyrush Including this header brings in all the examples.

Reply:
perfect, I missed this thanks!

Reply:
C'mon @cyrush get your act together! 😃